docs/getting_started_guide/Installation_Options/r10k/Preparing_For_Non_Rpm_Install.rst

```diff
@@ -13,10 +13,10 @@ from a fresh install:
 
```

Comment:
Line 9 says "alternative module path".  This is ambiguous.  I think 'secondary' module path would be more appropriate.  Also, the comment could talk about setting the primary and secondary environments in the environment.conf, where the primary module path contains modules and the secondary module path contains data that should not be managed by R10k.